contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt

```diff
@@ -485,6 +485,36 @@ open class ContourLayout(
         lambda = unwrapXIntLambda(provider)
     )
 
+  fun matchX(
```

Comment:
I think these should have docblocks. I haven't checked if the other entries in this file do, but we should get in the habit of thoroughly documenting the public API surface.